vueapp/components/Videos/Actions/VideoEmbeddingCode.vue

```diff
@@ -1,50 +1,55 @@
 <template>
     <div>
         <StudipDialog
-            :title="$gettext('Einbettungscode')"
+            :title="$gettext('Einbettungsoptionen')"
             :closeText="$gettext('Schließen')"
             :closeClass="'cancel'"
-            height="400"
+            height="450"
             width="550"
             @close="this.$emit('cancel')"
         >
             <template v-slot:dialogContent>
-                <textarea v-model="embeddingCode" rows="5" class="oc--embedding-code-text" readonly></textarea>
-
-                <StudipButton
-                    :disabled="!embeddingCode"
-                    @click.prevent="copyEmbeddingCode()"
-                >
-                    {{ $gettext('Einbettungscode kopieren') }}
-                </StudipButton>
-
+                <form class="default oc--video-actions-embedding">
+                    <label>
+                        {{ $gettext('Einbettungscode') }}
+                        <textarea :value="embeddingCode" rows="5" readonly></textarea>
+                    </label>
+                    <legend>
+                        {{ $gettext('Einbettungslink') }}
+                        <input type="text" :value="embeddingLink" readonly ref="embeddingLink" />
+                    </legend>
+                </form>
                 <MessageList :dialog="true" />
```

Comment:
This needs to be floated in order to not generating unwanted scroll!
```suggestion
                <MessageList :dialog="true" :float="true"/>
```